src/dotnet/Program.cs

```diff
@@ -114,6 +114,16 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 command = "help";
             }
 
+            var hashedargs = string.Join(",", args.Select(x => HashHelper.sha256_hash(x)));
```

Comment:
There is still an offline discussion regarding what arguments to capture and how to do it.